### ISSUE TITLE
stash error

### ISSUE THREAD

**Issue by Nippytime** [ChuckTheSheep/pz-community-modding#24]
noticing this always happens upon load.  tested with only this and cheat menu enabled in debug mode.

```
LOG  : General     , 1675106885137> 0> -------------------------------------------------------------
attempted index: size of non-table: null

LOG  : General     , 1675106885138> 0> -----------------------------------------
STACK TRACE
-----------------------------------------
function: populateList -- file: StashDebug.lua line # 72 | Vanilla
function: initialise -- file: StashDebug.lua line # 66 | Vanilla
function: OnOpen -- file: vanillaOpenPanels.lua line # 37 | MOD: Modding Community: Debug Tools
function: openOnStart -- file: vanillaOpenPanels.lua line # 85 | MOD: Modding Community: Debug Tools

ERROR: General     , 1675106885140> 0> ExceptionLogger.logException> Exception thrown java.lang.RuntimeException: attempted index: size of non-table: null at KahluaThread.tableget line:1689.
ERROR: General     , 1675106885140> 0> DebugLogStream.printException> Stack trace:
java.lang.RuntimeException: attempted index: size of non-table: null
	at se.krka.kahlua.vm.KahluaThread.tableget(KahluaThread.java:1689)
	at se.krka.kahlua.vm.KahluaThread.luaMainloop(KahluaThread.java:641)
	at se.krka.kahlua.vm.KahluaThread.call(KahluaThread.java:163)
	at se.krka.kahlua.vm.KahluaThread.pcall(KahluaThread.java:1980)
	at se.krka.kahlua.vm.KahluaThread.pcallvoid(KahluaThread.java:1812)
	at se.krka.kahlua.integration.LuaCaller.pcallvoid(LuaCaller.java:66)
	at se.krka.kahlua.integration.LuaCaller.protectedCallVoid(LuaCaller.java:139)
	at zombie.Lua.Event.trigger(Event.java:64)
	at zombie.Lua.LuaEventManager.triggerEvent(LuaEventManager.java:134)
	at zombie.gameStates.GameLoadingState.exit(GameLoadingState.java:405)
	at zombie.gameStates.GameStateMachine.update(GameStateMachine.java:105)
	at zombie.GameWindow.logic(GameWindow.java:298)
	at zombie.core.profiling.AbstractPerformanceProfileProbe.invokeAndMeasure(AbstractPerformanceProfileProbe.java:71)
	at zombie.GameWindow.frameStep(GameWindow.java:765)
	at zombie.GameWindow.run_ez(GameWindow.java:681)
	at zombie.GameWindow.mainThread(GameWindow.java:495)
	at java.base/java.lang.Thread.run(Unknown Source)
LOG  : General     , 1675106885141> 0> -----------------------------------------
STACK TRACE
-----------------------------------------
function: populateList -- file: StashDebug.lua line # 72 | Vanilla
function: initialise -- file: StashDebug.lua line # 66 | Vanilla
function: OnOpen -- file: vanillaOpenPanels.lua line # 37 | MOD: Modding Community: Debug Tools
function: openOnStart -- file: vanillaOpenPanels.lua line # 85 | MOD: Modding Community: Debug Tools
```

**Comment by ChuckTheSheep** [comment]
This occurs for me as well when testing in MP. I believe it is to do with server/client communication.

**Comment by Nippytime** [comment]
sounds good. it's literally nothing, but figured i'd use the github haha

**Comment by ChuckTheSheep** [comment]
Should be fixed with: c9230e0ae84fdbb350e18d7befbdb1f25b34c9f9
